sdk/communication/Azure.Communication.MediaComposition/tests/MediaCompositionClient/MediaCompositionClientLiveTests.cs

```diff
@@ -17,17 +17,17 @@ public class MediaCompositionClientLiveTests : MediaCompositionClientLiveTestBas
         {
             ["presenter"] = new ParticipantInput(
                 id: new MicrosoftTeamsUserIdentifier("f3ba9014-6dca-4456-8ec0-fa03cfa2b7b7"),
-                call: "teamsMeeting")
+                call: "acsGroupCall")
                 {
                    PlaceholderImageUri = "https://imageendpoint"
                 },
             ["support"] = new ParticipantInput(
                 id: new MicrosoftTeamsUserIdentifier("fa4337b5-f13a-41c5-a34f-f2aa46699b61"),
```

Comment:
Same here, update to CommunicationUser instead of MicrosoftTeamsUserIdentifier

Reply:
Updated all MicrosoftTeamsUserIdentifier to CommunicationUser. Pushed new changes.